.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -8,8 +8,8 @@ param
     $PullReqestJiraIssueDescription
 )
 
-$context = [string]::IsNullOrEmpty($IssueComponent) ? $GitHub.repository : $IssueComponent
-$titleSuffix = $SuffixIssueTitles ? " in $context" : ""
+$context = [string]::IsNullOrEmpty($IssueComponent) | Where-Object $GitHub.repository : $IssueComponent
+$titleSuffix = $SuffixIssueTitles | Where-Object " in $context" : ""
```

Comment:
These are ternary operators.

Reply:
Yeah, I didn't notice these - Invoke-Analyzers modified these with the -Fix switch.